pages/guides/build-a-cross-chain-voting-dapp/build-a-cross-chain-voting-dapp-part-1.mdx

```diff
@@ -0,0 +1,326 @@
+---
+title: "xCall tutorial part 1: Creating a cross chain voting dapp."
+date: 2023-07-27
+author: espanicon
+slug: x-call-tutorial-part-1-creating-a-cross-chain-voting-dapp
+description: xcall tutorial part 1, creating a cross chain voting dapp. Developing the smart contracts
+draft: false
+tags:
+- java
+- smart contract
+- solidity
+- xcall
+---
+## Introduction
+
+xCall is a standard interface to make permission-less calls between different blockchain networks. Each network that ICON is interoperable with has an xCall contract address that dApps can call to transfer data across chains.
```

Comment:
Because this is already stated in other parts of the docs prior to this point, this should be removed

Reply:
done